docs/docsite/extra-docs.yml

```
@@ -6,3 +6,6 @@ sections:
   - title: Module Development Guidelines
     toctree:
       - dev_guidelines
+  - title: Dynamic Inventory Plugin Guidelines
```

```suggestion
  - title: Dynamic Inventory Plugin Guide
```

It's a guide (how something *can* be done) rather than guidelines (how something *should*/must be done)